server/routerlicious/packages/services-telemetry/src/lumberjack.ts

```diff
@@ -20,33 +35,51 @@ import {
 export class Lumberjack {
 	private readonly _engineList: ILumberjackEngine[] = [];
 	private _schemaValidators: ILumberjackSchemaValidator[] | undefined;
+	private _options: ILumberjackOptions = defaultLumberjackOptions;
```

Comment:
I am not sure I understand why we need an `_options` besides the static `_staticOptions`. We use `_staticOptions` to return the global Lumberjack instance if the setting `enableGlobalTelemetryContext` is enabled.

For the instance-level options, we use it in line 186 to add properties to the current lumberjack log if there are global properties available. But couldn't we also be using the static `_staticOptions` in that case?

Reply:
We _could_ use `_staticOptions` everywhere, however, I think that would be bad design because it would alter the behavior of specific instances based on the global, static instance. For example, if we wanted to have a special, local instance of Lumberjack to do something specific, we might want to configure it in a different way than the global instance.

Reply:
Ah, got it! I missed the fact that we could update the `_options` property of each instance and was under the impression it would only be set once in the beginning. I agree with your idea